src/content/docs/integrations/react-native.mdx

```diff
@@ -108,3 +108,9 @@ For end-to-end testing, make sure you have [Enabled MSW in development](#develop
 -import { setupServer } from 'msw/node'
 +import { setupServer } from 'msw/native'
 ```
+
+### Network request fails with `TypeError: Failed to fetch`
+
+**Reason:** `/node_modules/@mswjs/interceptors/src/interceptors/fetch/index.ts` throws at `createNetworkError` because your `Node` version is not `≥18.16`
```

Comment:
This is the immediate reason but not the underlying reason. 

Did you manage to figure out what exactly was broken in Node.js <18.16? We need to list _that_ as the actual reason. The Interceptors library doesn't have any logic that would call `createNetworkErorr()` if you use it in different versions of Node.js. It's a generic wrapper around runtime errors. Let's dive deeper into the error thrown and provide the actual reason.

Reply:
Valid comment, will look into it.

Reply:
Note that the `Failed to fetch` error has a `.cause` property that includes the original error thrown. That's a good place to start.